SKILL.tsv

```diff
@@ -5201,9 +5201,9 @@ SKILL_20150918_005200	$Increases the damage dealt on an enemy with [High Kick] b
 SKILL_20150918_005201	$Earth Wave: Remove Knockdown
 SKILL_20150918_005202	$[Earth Wave] no longer knockdowns enemies. (Only applies if the attribute is set to ON)
 SKILL_20150918_005203	$Steed Charge: Remove Knockdown
-SKILL_20150918_005204	$[Steed Charge] no longer knockdowns enemies. (Only applies if the attribute is set to ON)
+SKILL_20150918_005204	$[Steed Charge] no longer knocks enemies down. (Only applies if the attribute is set to ON)
 SKILL_20150918_005205	$Doom Spike
-SKILL_20150918_005206	$[Doom Spike] no longer knockdowns enemies. (Only applies if the attribute is set to ON)
+SKILL_20150918_005206	$[Doom Spike] no longer knocks enemies down. (Only applies if the attribute is set to ON)
```

Comment:
feel like it could be either way. 
[Doom Spike] will no longer knock enemies down.
[Doom Spike] no longer knocks enemies down.

If this is a toggle or something and it gets toggled off, I can imagine the message would include 'will' when sending a message to the player alerting them that it won't knock down anymore. Up to you.